components/costs/cost-usage-report.yaml

```diff
@@ -0,0 +1,43 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+Description: setup cost usage report
+Resources:
+  CostBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: superwerker-cost-usage-report
```

Comment:
We cannot use static bucket names (since these are geo global)

Reply:
is it ok to pass the name as parameter?

Reply:
Just leave the attribute out - it does not generated value for the user to be able to specify a name I think. It should be part of the `outputs` though.

Reply:
done